src/main/java/it/gov/pagopa/pu/debtpositions/exception/custom/TooManyElementsException.java

```diff
@@ -0,0 +1,5 @@
+package it.gov.pagopa.pu.debtpositions.exception.custom;
+
+public class TooManyElementsException extends RuntimeException{
```

Comment:
There is already a TooManyRecordsExpection on JPA which is thrown when you are asking for just 1 record, but the query found more than 1
```suggestion
public class ExportTooManyRecordsException extends RuntimeException{
```